lacecore/_obj.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
```

Comment:
What do you think would be the public interface for calling this? One option could be:

```py
from lacecore import load_obj
load_obj('path/to.obj')
```

Reply:
LGTM

Reply:
This is done.